### ISSUE TITLE
Make shotObstacle a dynamic obstacle so robots can still go through before and after the shot.

### ISSUE THREAD

**Issue by ashaw596** [RoboJackets/robocup-software#875]
Currently we just block off giant parts of the screen and almost crash our pathplanner due to RRT failures.

**Comment by jgkamat** [comment]
To make this dynamic we need to know when we kick, which might be a problem figuring out. We could fudge this, but idk.

**Comment by ashaw596** [comment]
Yeah, the idea would be that we should know when we're going to kick. I'm halfway there on the line kick function and its easy to modify pivot kick to do this.
